src/cryptoadvance/specter/templates/includes/wallet_menu.html

```diff
@@ -0,0 +1,7 @@
+<div class="row padded">
+	<a href="{{ url_for('wallet_tx',wallet_alias=wallet_alias)}}" class="btn radio left {% if active_menu_item == 'transactions' %}checked{% endif %}">Transactions</a>
+	<a href="{{ url_for('wallet_addresses',wallet_alias=wallet_alias)}}" class="btn radio {% if active_menu_item == 'addresses' %}checked{% endif %}">Addresses</a>
+	<a href="{{ url_for('wallet_receive',wallet_alias=wallet_alias)}}" class="btn radio {% if active_menu_item == 'receive' %}checked{% endif %}">Receive</a>
+	<a href="{{ url_for('wallet_send',wallet_alias=wallet_alias)}}" class="btn radio {% if active_menu_item == 'send' %}checked{% endif %}">Send</a>
+	<a href="{{ url_for('wallet_settings',wallet_alias=wallet_alias)}}" class="btn radio right {% if active_menu_item == 'settings' %}checked{% endif %}">Settings</a>
```

Comment:
Here you compare against `active_menu_item` but everywhere you set `active_menuitem`
So active item is never highlighted